### ISSUE TITLE
np.genfromtxt time consumption

### ISSUE THREAD

**Issue by VOMAY** [scivision/georinex#26]
Dear Scivision,

I put your latest release v1.6.1 in a profiler. After getting rid of xarrax.concat and .merge, the new bottle neck is the 'np.genfromtxt' function:
``` python
darr = np.genfromtxt(BytesIO(raws.encode('ascii')), delimiter=[Lf, 1, 1]*hdr['Nobs'])
```
I know np.genfromtxt is handy, but not the fastest. Did you checkout `pandas.read_csv` or `pandas.read_fwf`? Alternatively, a specialized self-written read function may be quickest. I think it is worth spending some extra thoughts on this one, since it is the center-piece of the whole software.

**Comment by scivision** [comment]
The current `master` commit has implemented the full ~20x speedup over v1.6.0, which is an additional 2x speedup over v1.6.1, by using a pure Python implementation instead of np.genfromtxt

**Comment by VOMAY** [comment]
I see you actually changed quite a lot in the code to achieve this, but you managed to improve the speed by moving away from np.genfromtxt.

Thanks for the update!
